sections/05_treat.md

```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
+predicting which cases will progress to dementia, they were able to
+efficiently assign patients to a clinical trial and reduced the required
+sample sizes by a factor of five.  Similarly, Artemov et al
+[@doi:10.1101/095653] applied deep learning to predict which clinical trials
+were likely to fail and which were likely to succeed. By predicting the side
+effects and pathway activations of each drug, and then translating these
+activations to a success proability, their deep learning-based approach was
+able to significantly outperform a random forest classifier trained on gene
+expression changes. These approaches suggest promising directions to improve
+the efficiency of clinical trials and accelerate drug development.
+
+#### Challenges
+
+##### Actionable Interpretability
+
+A common challenge in many applied deep learning problems is the consideration
+of deep learning models as uninterpretable "black boxes". Without human-
+intelligible reasoning for the model's predictions, it is difficult to trust
+the model. This presents a major challenge for the risk-averse task of
+clinical decision making. As described above, there has been some work to
+directly assign treatment plans without interpretability; however, the removal
+of human experts from the decision-making loop make the models difficult to
+integrate with clinical practice. To alleviate this challenge, several studies
+have attempted to create more interpretable deep models, either specifically
+for healthcare or as a general procedure for deep learning.
+
+For domain-specific models, studies have primarily focused on integrating
+attention mechanisms with the neural networks. Attention mechanisms
+dynamically weight the importance the neural network gives to each feature. By
+inspecting the attention weights for a particular sample, a practitioner can
+identify the important features for a particular prediction. Choi et al
+[@arxiv:1608.05745] inverted the typical architectue of recurrent neural
+networks to improve interpretability. In particular, they only used recurrent
+connections in the attention generating procedure, leaving the hidden state
+directly connected to the input variables. This model was able to produce
+accurate diagnoses in which the contribution of previous hospital visits could
+be directly interpreted. Choi et al [@arxiv:1611.07012] later extended this
+work to take into account the structure of disease ontologies and found that
+the concepts represented by the model were aligned with medical knowledge. Che
+et al [@arxiv:1512.03542] introduced a knowledge-distillation approach which
+used gradient boosted trees to learn interpretable healthcare features from
+trained deep models.
+
+As this challenge of interpretability is common across many domains, there is
+significant interest in developing generic procedures for knowledge extraction
+from deep models. Ribeiro et al [@arxiv:1602.04938] focus on interpreting
```

Comment:
Unsurprisingly, interpretability has come up in multiple sections.  I suggest that we keep this paragraph here for now and merge as-is.  We can consider moving it to the Discussion once we review the entire manuscript in the editing phase.  In that case, we could refer to the Discussion section here.

I'm glad you covered #280 and #308.  These had been on my TODO list.